array-manipulation-1/src/head.js

```diff
@@ -0,0 +1,9 @@
+/* exported head */
+// make storage for first element of the array
+// get the first element of array and store it
+// give back location of first element stored
+function head(array) {
+  var firstIndex = [];
```

Comment:
Making this variable with a default value of an empty array is unnecessary, you can just save the value you want in it when you declare the variable.